terraform/Makefile

```diff
@@ -28,7 +28,9 @@ endif
 export OS_CLOUD ?= $(ENVIRONMENT)
 
 export TF_VAR_cloud_provider=$(ENVIRONMENT)
-export TF_CLI_CONFIG_FILE=terraformrc
+ifneq (,$(wildcard terraformrc))
+  export TF_CLI_CONFIG_FILE ?= terraformrc
```

Comment:
Do we need this at all? Maybe we can rather just drop setting this variable?

Reply:
We also have [this](https://github.com/osism/testbed/blob/main/terraform/terraformrc.sample) file. Should be dropped than as well. I have no strong opinion on this topic. @berendt what do you think?

Reply:
I would continue to support the file itself and not remove it completely. Sometimes you need the terraformrc to customize things.

Reply:
let's keep it then.